README.md

```diff
@@ -29,12 +29,19 @@ It is possibile to snooze the notification issue for `x` time by closing it as *
 
 **Optional** A GitHub token.
 
-### `stale-days: 7`
+### **Deprecated** `stale-days: 7`
 
 **Optional** The time after which unreleased commits should be considered stale and should notify for a release. The value can be a number or a string.
 Example: `1 minute`.
 Default is `7` days.
 
+### `notify-after: '7 days'`
+
+**Optional** The time after which unreleased commits should be considered stale and should notify for a release.
+This option accepts various time formats as described by the time conversion [library](https://github.com/vercel/ms) used internally.
```

Comment:
docs: putting "library" as the link text is not great. use "ms" as the link text so it's obvious to a user what library it is without having to click the link